File: src/main/java/guru/xgm/language/zinc/psi/ZincTokenSets.java
package guru.xgm.language.zinc.psi;

import com.intellij.psi.tree.TokenSet;

import static guru.xgm.language.zinc.psi.ZincTypes.*;

public interface ZincTokenSets {
    TokenSet COMMENTS = TokenSet.create(LINE_COMMENT, BLOCK_COMMENT);
}

File: src/main/kotlin/guru/xgm/language/zinc/psi/ZincTokenSets.kt
package guru.xgm.language.zinc.psi

import com.intellij.psi.tree.TokenSet

interface ZincTokenSets {
    companion object {
        @JvmField
        val COMMENTS: TokenSet = TokenSet.create(ZincTypes.LINE_COMMENT, ZincTypes.BLOCK_COMMENT)
    }
}
